README.md

```diff
@@ -15,15 +15,22 @@ pip install git+https://github.com/jellyfish-dev/python-server-sdk
 
 ## Usage
 
-First create a `RoomApi` instance, providing the jellyfish server address and api token
+The SDK exports two main classes for interacting with Jellyfish server:
+`RoomApi` and `Notifier`.
+
+`RoomApi` wraps http REST api calls, while `Notifier` is responsible for receiving real-time updates from the server.
+
+#### RoomApi
+
+Create a `RoomApi` instance, providing the jellyfish server address and api token
 
 ```python
 from jellyfish import RoomApi
 
-room_api = RoomApi(server_address='http://localhost:5002', server_api_token='development')
+room_api = RoomApi(server_address='localhost:5002', server_api_token='development')
 ```
 
-You can use it to interact with Jellyfish managing rooms, peers and components
+You can use it to interact with Jellyfish, managing rooms, peers and components
```

Comment:
```suggestion
You can use it to interact with Jellyfish, manage rooms, peers and components
```
or `...Jellyfish, enabling the management of rooms...`, now it sounds kinda weird.